package.json

```diff
@@ -38,7 +38,6 @@
     "dotenv": "^4.0.0",
     "istanbul": "^0.4.5",
     "mocha": "^3.2.0",
-    "mongoose-findorcreate": "^2.0.0",
```

Comment:
Removed this dependency as this is unmerged, so no support for mongoose 5 yet. https://github.com/drudge/mongoose-findorcreate/pull/18